MediaPlayer/MediaPlayer.js

```diff
@@ -331,7 +352,8 @@ const MediaPlayerBehaviorDecorator = hoc((config, Wrapped) => { // eslint-disabl
 				currentTime: el.currentTime,
 				duration: el.duration,
 				loop: el.loop,
-				paused: el.paused
+				paused: el.paused,
+				proportionPlayed: el.proportionPlayed
```

Comment:
Let's give a fallback value as 0.
```suggestion
				proportionPlayed: el.proportionPlayed || 0
```

Reply:
thanks. the change is pushed